docs/documentation/server_admin/topics/organizations/authenticating-members.adoc

```diff
@@ -0,0 +1,99 @@
+
+If the realm has the self-registration setting enabled, the user can click on the `Register` link at the identity-first login page
+and create an account at the realm.
+After that, the administrator can send an invitation link to the user or manually add the user as a member of an organization.
+Look at the <<_managing_members_,Managing members>> for more details.
+
+If the organization has an identity provider without a domain set, and they are marked as public, they can also click on
+the identity provider link at the identity-first login page to automatically create an account and join an organization
+once they authenticate through the identity provider.
+Look at the <<_managing_identity_provider_,Managing identity providers>> for more details.
+
+Similar to the above, if the organization has an identity provider set with one of the organization domains,
+the user will be automatically redirected to the identity provider if their email matches a specific domain to authenticate
+and automatically create an account and join an organization once the flow is completed.
+
+== Configuring existing authentication flows
+
+As previously mentioned, for new realms authentication flows are automatically updated with the necessary steps
+to support organizations and authenticate their members. For existing realms, in addition to enabling organizations to the
+realm, you also need to manually update your existing (custom) authenticating flows.
+
+Change the `browser` flow by following these steps:
+
+.Procedure
+. Duplicate the current flow bound to the `Browser flow` binding type to avoid breaking the flow you are currently using
+. Click on `Add sub-flow` and give it a name such as `My Organization`
+. Move the newly added `My Organization` sub-flow to execute right after the `Identity Provider Redirector` execution step.
+The main point here is that the sub-flow should happen before any other sub-flow or execution step that authenticates the
+user using whatever credentials you support in your realm. Once added, change the `Requirement` to `Alternative`.
+. Click on `Add sub-flow` in the `My Organization` sub-flow and give it a name such as `My Organization - Conditional`. Once added, change the `Requirement` to `Conditional`.
+. Click on `Add condition` in the `My Organization - Conditional` sub-flow and select `Condition - user configured`. Once added, change the `Requirement` to `Required`.
+. Click on `Add step` in the `My Organization - Conditional` sub-flow and select the `Organization Identity-First Login
+` execution step. Once added, change the `Requirement` to `Alternative`.
+. Bind the authentication flow to the `Browser` binding type.
+
+Once you enable the <<_enabling_organization_,Organizations>> setting to the realm and create
+at least a single organization, you should be able to see the identity-first login page and start using organizations
+in your realm.
+
```

Comment:
Use `backticks` only for what you type or enter
Use *bold* for whatever appears in the UI.

Reply:
Changed here https://github.com/keycloak/keycloak/pull/29796/commits/b40f57be332a46f8648dac83d3cdfc8926ef5e6f.